SafeExamBrowser.SystemComponents.Contracts/ISystemInfo.cs

```diff
@@ -47,5 +47,10 @@ public interface ISystemInfo
 		/// The MAC Addres of the network addapter
```

Comment:
It would be great if you could fix the typos in the comment ("The MAC address of the network adapter.").